Classes/Configuration/ExtConf.php

```diff
@@ -23,37 +22,20 @@
 #[Autoconfigure(constructor: 'create')]
 final class ExtConf
 {
-    private const EXT_KEY = 'ext_kickstarter';
-
-    private const DEFAULT_SETTINGS = [
-        // general
-        'exportDirectory' => '',
-        'activateModule' => false,
-    ];
-
     public function __construct(
         // general
-        private string $exportDirectory = self::DEFAULT_SETTINGS['exportDirectory'],
-        private bool $activateModule = self::DEFAULT_SETTINGS['activateModule'],
+        private string $exportDirectory = Settings::DEFAULT_SETTINGS['exportDirectory'],
+        private bool $activateModule = Settings::DEFAULT_SETTINGS['activateModule'],
     ) {}
 
     public static function create(ExtensionConfiguration $extensionConfiguration): self
     {
-        $extensionSettings = self::DEFAULT_SETTINGS;
-
-        // Overwrite default extension settings with values from EXT_CONF
-        try {
-            $extensionSettings = array_merge(
-                $extensionSettings,
-                $extensionConfiguration->get(self::EXT_KEY),
-            );
-        } catch (ExtensionConfigurationExtensionNotConfiguredException|ExtensionConfigurationPathDoesNotExistException) {
-        }
+        $extensionSettings = Settings::getSettings($extensionConfiguration);
```

Comment:
wow, wow. What is happening here? This ExtConf thingy was a long discussion with Stefan Bürk. It was his idea to use AutoConfiguration and having "create" as static while having the constructor for tests was an awesome idea.
Testing static declared methods is no fun, and I would please you to prevent such code wherever possible.

Reply:
yeah true, some left over. Removed.